src/test/java/ulaval/glo2003/utils/ProductUtils.java

```diff
@@ -0,0 +1,209 @@
+package ulaval.glo2003.utils;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import ulaval.glo2003.api.product.ProductCollectionResponse;
+import ulaval.glo2003.api.product.ProductRequest;
+import ulaval.glo2003.api.product.ProductResponse;
+import ulaval.glo2003.domain.product.Product;
+import ulaval.glo2003.domain.product.ProductCategory;
+import ulaval.glo2003.domain.product.ProductFilter;
+import ulaval.glo2003.domain.product.SaleStatus;
+
+public class ProductUtils {
```

Comment:
Pourquoi juste `Utils` ? C'est pas un utilitaire pour les produit, mais pour les tests de produit, le Test était important. même chose pour les autres 

Aussi, plus un pet-peeve que d'autre chose, mais pas besoin de mettre Product dans tous les noms de fonction, on est dans une classe nommé `Product...`, on s'attend pas à recevoir une Request d'un autre type (Spécifier collection or not est bon though)